portfolio/src/main/webapp/html/pages/Map.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Map</title>
+    <link rel="stylesheet" href="../../style.css">
+    <script src="/javascript/script.js"></script>
+  </head>
+  <body>
+    <iframe class="navbar" src="../components/navbar.html"></iframe>
+    <h1 class="center">Map</h1><br>
+    <iframe id="map-frame" src="../components/map.html"></iframe>
```

Comment:
Is it possible to combine Map.html and /components/map.html into one file? I think it would be easier to read if most of the map-related html was in one place.

Reply:
I tried but when I put the map on the actual page it wouldn't show up, I don't know why. It was the exact same code